exchange/adapter_builders.go

```diff
@@ -150,6 +151,7 @@ func newAdapterBuilders() map[openrtb_ext.BidderName]adapters.Builder {
 		openrtb_ext.BidderAdtarget:          adtarget.Builder,
 		openrtb_ext.BidderAdtelligent:       adtelligent.Builder,
 		openrtb_ext.BidderAdvangelists:      advangelists.Builder,
+		openrtb_ext.BidderAdview:    	     adview.Builder,
```

Comment:
It looks like your company name is "AdView". You are welcomed to rename `BidderAdview` to `BidderAdView` to better match your brand.